manim/mobject/coordinate_systems.py

```diff
@@ -620,22 +697,26 @@ def get_lines(self):
         return lines1, lines2
 
     def get_lines_parallel_to_axis(
-        self, axis_parallel_to, axis_perpendicular_to, freq, ratio_faded_lines
-    ):
+        self,
+        axis_parallel_to: Line,
+        axis_perpendicular_to: Line,
+        freq: float,
+        ratio_faded_lines: float,
```

Comment:
You've set the type hint for parameter `faded_line_ratio` as `int` in the constructor (see previous comment) but here you've set it to `float`. Please resolve this.